enhancements/olm/report-sdk-operators.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Enable tracking of SDK operators in OLM metrics
+authors:
+  - "@varshaprasad96"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-05-28
+status: implementable
+---
+
+# Enable tracking of SDK operators in OLM metrics
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+This enhancement proposes the addition of operator specific labels to existing OLM metrics, in order to collect data on Redhat operators which utilize SDK to build image bundle.
+
+## Motivation
+
+Collecting metrics regarding the usage of Operator SDK among operator authors is important to measure the reachability of the tool. Since OLM already has a pipeline built to obtain metrics on operators and report it to the RedHat telemetry, we intend to utilize the same and obtain data on RedHat operators (installed on OpenShift namespaces) using SDK to build bundle images.
```

Comment:
```suggestion
Collecting metrics regarding the usage of Operator SDK among operator authors is important to measure the reachability of the tool. Since OLM already has a pipeline built to obtain metrics on operators and report it to the Red Hat OpenShift telemetry, we intend to utilize the same and obtain data on Red Hat operators (installed in OpenShift namespaces) using SDK to build bundle images.
```